packages/engine-test-utils/src/__tests__/engine-server/enginev2.spec.ts

```diff
@@ -54,6 +58,55 @@ describe("engine, cache", () => {
     );
   });
 
+  test("delete stale entries from cache", async () => {
```

Comment:
can we have some more test cases?
ideally, add this to the `delete` preset > https://github.com/dendronhq/dendron/blob/chore/migrate-cal-widget-kevin/packages/engine-test-utils/src/presets/engine-server/delete.ts#L38:L38

make sure the cache is correct after the deletion

Reply:
Moved and updated other presets to test for cache